content/pt-br/docs/concepts/policy/node-resource-managers.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gerenciamento de Recursos de Nós
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Para dar suporte a latência crítica e a cargas de trabalho de alta taxa de transferência, o kubernets oferece um conjunto de gerenciadores de recursos. Os gerenciadores visam coordenar e otimizar o alinhamento de recursos do nó(s) para Pods configurados com um requisito específico para CPUs, dispositivos e recursos de memória (*hugepages*).
+
+<!-- body -->
+
+O gerenciador principal, o Gerenciador de Topologia, é um componente do Kubelet que coordena o processo geral de gerenciamento de recursos por meio da sua [política](/docs/tasks/administer-cluster/topology-manager/).
+
+A configuração de gerenciadores individuais é elaborada em documentos dedicados:
```

Comment:
nit: original term was "elaborated", but it carries different meanings in English (present in detail). How do you feel about replacing "elaborada" here with "discutida"?

Reply:
Seems to make more sense with "discutida", I'll switch